dtos/user_dto.go

```diff
@@ -1,9 +1,9 @@
 package dto
 
 type CreateUser struct {
-	UserName  string `json:"username"`
-	Name      string `json:"name"`
-	PublicKey string `json:"publicKey"`
+	UserName     string `json:"username"`
+	Name         string `json:"name"`
```

Comment:
Not sure we need a Name to create a user. We can let the invited user fill in his name

Reply:
possible hack where the user changes the name to confuse other users. intern changing name to cto